src/main/markdown/doc/latest/FAQ_UI.md

```diff
@@ -7,7 +7,7 @@ FAQ - UI
     3.  [Why doesn't setting a widget's height as a percentage work?](#Why_doesn)
     4.  [Does GWT support standards mode?](#Does_GWT_support_standards_mode?) 
 2.  [Event Handling and Performance](#Event_Handling_and_Performance)
-    1.  [How do I display a big list of items in a table with good performance?](#How_do_I_display_a_big_list_of_items_in_a_table_with_good_perfor)
+    1.  [How do I display a big list of items in a table with good performance?](#How_do_I_display_a_big_list_of_items_in_a_table_with_good_perform)
```

Comment:
This is a weird one - at a glance some tool might have previously said that 64 chars is the max for an anchor, why just add one char if we think that's good enough? line 13 could get to be "slowly" then, etc.

This isn't visible to users, i'd say skip all else equal?